src/components/Time.tsx

```diff
@@ -0,0 +1,32 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+
+import React, { useState, useEffect } from "react";
+
+export default function Time() {
+  const [currentTime, setCurrentTime] = useState(new Date());
+
+  const hours = currentTime.getHours();
+  const minutes = currentTime.getMinutes();
+  const seconds = currentTime.getSeconds();
+
+  useEffect(() => {
+    setInterval(() => {
+      setCurrentTime(new Date());
+    }, 0);
+  }, []);
```

Comment:
```suggestion
    const interval = setInterval(() => {
      setCurrentTime(new Date());
    }, 0);
    
    return () => { clearInterval(interval); }
  }, []);
```

위 컴포넌트에서 setTimeout을 클리어하신 것처럼 인터벌도 클리어하실 수 있어요!

useEffect의 반환되는 콜백은 컴포넌트 언마운트시에 실행되는 것도 알고 계시면 좋구요!